src/components/views/donate/DonateIndex.tsx

```diff
@@ -97,39 +105,122 @@ const ProjectsIndex = (props: IProjectBySlug) => {
 		);
 	};
 
+	const SocialBox = (props: any) => {
+		return (
+			<Social isSuccess={props.isSuccess}>
+				<Lead>
+					{props?.isSuccess
+						? 'Share this with your friends'
+						: `Can't donate? Share this page instead.`}
+				</Lead>
+				<SocialItems>
+					<SocialItem isSuccess={props.isSuccess}>
+						<TwitterShareButton
+							title={shareTitle}
+							url={url || ''}
+							hashtags={['Giveth']}
+						>
+							<Image
+								src={'/images/social-tw.svg'}
+								alt='tw'
+								width='44px'
+								height='44px'
+							/>
+						</TwitterShareButton>
+					</SocialItem>
+					<SocialItem isSuccess={props.isSuccess}>
+						<LinkedinShareButton
+							title={shareTitle}
+							summary={project?.description}
+							url={url || ''}
+						>
+							<Image
+								src={'/images/social-linkedin.svg'}
+								alt='lin'
+								width='44px'
+								height='44px'
+							/>
+						</LinkedinShareButton>
+					</SocialItem>
+					<SocialItem isSuccess={props.isSuccess}>
+						<FacebookShareButton
+							quote={shareTitle}
+							url={url || ''}
+							hashtag='#Giveth'
+						>
+							<Image
+								src={'/images/social-fb.svg'}
+								alt='fb'
+								width='44px'
+								height='44px'
+							/>
+						</FacebookShareButton>
+					</SocialItem>
+				</SocialItems>
+			</Social>
+		);
+	};
+
 	const SuccessView = () => {
 		return (
 			<SucceessContainer>
-				<H4 color={semanticColors.jade[500]}>Successfully donated</H4>
-				<Image
-					src='/images/motivation.svg'
-					alt='motivation'
-					width='121px'
-					height='121px'
-				/>
-				<SuccessMessage color={brandColors.deep[900]}>
-					We have received your donation, you can see this project on
-					your account under the Donated projects and follow the
-					project updates there or take a shortcut here. Go to
+				<ConfettiContainer>
+					<ConfettiAnimation size={300} />
+				</ConfettiContainer>
+				<H4 color={semanticColors.jade[500]}>You're a giver now!</H4>
```

Comment:
The H4 component didn't accept color as a prop. please change the color with styled-component.